samples/standalone/src/main/kotlin/org/jetbrains/jewel/samples/standalone/Main.kt

```diff
@@ -27,6 +28,8 @@ import org.jetbrains.jewel.window.styling.TitleBarStyle
 import java.io.InputStream
 
 fun main() {
+    JewelLogger.getInstance("JewelStandAloneShowcase").warn("Starting Jewel Standalone showcase app")
```

Comment:
```suggestion
    JewelLogger.getInstance("JewelStandaloneShowcase").info("Starting Jewel Standalone showcase app")
```

We always style it as "standalone", one word. Also doesn't look like a warning